test/distributed/fsdp/test_fsdp_comm.py

```diff
@@ -0,0 +1,149 @@
+# Owner(s): ["oncall: distributed"]
```

Comment:
nit: could we rename this test file name to be "test_fsdp_no_sync_comm" or merge with "test_fsdp_no_sync"?  as both of them are testing no_sync features